packages/contents/src/drivefs.ts

```diff
@@ -310,6 +310,10 @@ export class DriveFSEmscriptenNodeOps implements IEmscriptenNodeOps {
           }
           break;
         }
+        case 'ctime':
+        case 'dontFollow':
+          // Ignore for now
+          break;
```

Comment:
This appears to be making the UI tests pass again.

Probably Firefox was also crashing in the UI tests, caused by all the warning messages logged in the console.